Magitek/Logic/Warrior/Buff.cs

```diff
@@ -41,10 +41,13 @@ public static async Task<bool> InnerRelease()
             if (!WarriorSettings.Instance.UseInnerRelease)
                 return false;
 
-            if (Combat.Enemies.Count(r => r.Distance(Core.Me) <= 3 + r.CombatReach) < 1)
+            //Radius of WAR AoEs is 5y not 3y, adding combat reach should be okay
```

Comment:
I assume this is 3y because it's checking for an enemy being in range for fell cleave, not for AOE. I don't think this is a bad change necessarily since the player is probably going to move into range, but if for some reason they can't then this would cause a wasted IR.